README.md

```diff
@@ -1,47 +1,75 @@
-# conu
-__CON__ -tainers __U__ -tils
-
-It provides low level __API__ for docker image/container testing.
-It is based on bash commands because primary target is to __test it
-as customers will__ use it.
-That's why it does not use python docker bindings.
-It provides python classes for:
-- __docker__ module
-  - __Image__ - image handling
-  - __Container__ - docker container handling
-- __utils__ module
-  - __Volume__ - it is mktemd utility on steroids
-  - __Probe__ - provides, simple port/file checks and waiting for this events
+# CONU
```

Comment:
`start picking-nits`

Can we agree on preferred form? I usually refer to the project as `conu`. I can see that used various forms: Conu, CONU, ConU. I would love if just used a single one. Which one do you guys prefer?

Reply:
I think I like all caps most, since technically it is an acronym.

Reply:
@dhodovsk this is exactly the reason why I like ConU most ;)

Reply:
3 people = 3 preferences #naming-is-hard

Reply:
For me CONU -> ConU.

@TomasTomecek what about your preference?

Reply:
I'm a minimalist, so...

> I usually refer to the project as `conu`.

Reply:
I vote for conu. IMO it's always good to have the project name same as the importable module - it makes things easier (and also I don't like capitals in project name).

Reply:
I vote for conu. ConU does not look from my point of view. My vote has strong like two times;)